### DISCUSSION TITLE
Please help me

### DISCUSSION THREAD

**Discussion by firdausiregar** [community/community#135494] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have submitted a support ticket on the support.github.com page but it has been 2 weeks without an answer, I want to ask why my account was banned

**Comment by ebndev** [comment]
Hi @firdausiregar,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor look into the details of your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
